src/components/radio/RadioButton.tsx

```diff
@@ -1,33 +1,35 @@
 import React from 'react';
 import styled, { css } from 'styled-components';
 import { colors } from 'common/colors';
+import { RadioProps } from './RadioGroup';
 
-interface RadioProps {
-  labels: string[];
-  groupName: string;
-  disabled?: boolean;
-  error?: boolean;
+interface RadioButtonProps extends Omit<RadioProps, 'choices' | 'onChange'> {
+  value: string;
+  children: React.ReactNode;
```

Comment:
Is this the right type? I would assume it would be some sort of collection of React.ReactNode?

Reply:
`React.ReactNode` is the default type for` children`, yes.
`React.ReactNode` can be _almost_ anything, both a collection of things, but also a single element, function, string, etc.
The type for `children` is usually inferred from `React.FC<Props>` which adds the `children`-prop.
I could perhaps remove the definition and use `React.FC<Props>` , but we've had some issues 
with Storybook not picking up the props with that